packages/types/src/runner-config.ts

```diff
@@ -45,6 +45,11 @@ export type KubernetesSequenceConfig = CommonSequenceConfig & {
     type: "kubernetes"
 }
 
-export type SequenceConfig = DockerSequenceConfig | ProcessSequenceConfig | KubernetesSequenceConfig
+export type SequenceConfigDisplay = CommonSequenceConfig & {
+    type: string
```

Comment:
`type: string` already exist in `CommonSequenceConfig` so this new type doesn't define any new type... `SequenceConfigDisplay=CommonSequenceConfig`

Reply:
You are right, I missed the fact that types like dockerconfig, kubernetsconfig overwrites the "type" field with the const. My bad.